atlp-rwanda/commanders-rn-medica
# Fix:Onboarding screens

What does this PR do?
This PR fixes how the onboarding screens were implemented where they were appearing everytime someone opens the app which is not the correct way they should only appear when someone visits the app for the first time 

Description of Task to be completed?
use redux to fix the issue of onboarding screens appearing everytime a person opens the app

How should this be manually tested?
Open the app and you will see that onboarding screens are fixed and they will be displaying only once 

Any background context you want to provide?


What are the relevant pivotal tracker/Trello stories?
N/A
